ballerina/googlesheets_client.bal

```diff
@@ -31,21 +31,21 @@ type RowValues record {
 
 # The client used by the generated persist clients to abstract and
 # execute API calls that are required to perform CRUD operations.
-public client class GoogleSheetsClient {
+public isolated client class GoogleSheetsClient {
 
     private final sheets:Client googleSheetClient;
     private final http:Client httpClient;
-    private string spreadsheetId;
-    private int sheetId;
-    private string entityName;
-    private string tableName;
-    private string range;
-    private map<SheetFieldMetadata> fieldMetadata;
-    private map<string> dataTypes;
-    private string[] keyFields;
-    private function (string[]) returns stream<record {}, Error?>|Error query;
-    private function (anydata) returns record {}|NotFoundError queryOne;
-    private map<function (record {}, string[]) returns record {}[]|error> associationsMethods;
+    private final string spreadsheetId;
+    private final int & readonly sheetId;
+    private final string entityName;
+    private final string tableName;
+    private final string range;
+    private final map<SheetFieldMetadata> & readonly fieldMetadata;
+    private final map<string> & readonly dataTypes;
+    private final string[] & readonly keyFields;
+    private isolated function (string[]) returns stream<record {}, Error?>|Error query;
+    private isolated function (anydata) returns record {}|NotFoundError queryOne;
+    private final (map<isolated function (record {}, string[]) returns record {}[]|Error>) & readonly associationsMethods;
```

Comment:
The type in https://github.com/ballerina-platform/module-ballerina-persist/blob/e4aaa4af015a8e076f4d5bee648f3eb2e5a7456f/ballerina/googlesheets_client.bal#L343 needs to be updated to include `isolated` too.